src/hooks/useCivicProfile.js

```diff
@@ -10,18 +10,83 @@ import { RDF_PREDICATES } from '@constants';
 import useSession from './useSession';
 import useRdfCollection from './useRdfCollection';
 
+const prefix = 'urn:hud:hmis:owl#';
+
+const civicProfileConfig = {
+  firstName: {
+    type: 'string',
+    predicate: 'urn:hud:hmis:owl#FirstName'
+  },
+  lastName: {
+    type: 'string',
+    predicate: 'urn:hud:hmis:owl#LastName'
+  },
+  dateOfBirth: {
```

Comment:
Think I'll start refactoring DOB out of what we have now in the public profile in another PR.